pkgs/tools/misc/sd-mux-ctrl/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, fetchgit, cmake, pkgconfig, libftdi1, popt}:
+
+stdenv.mkDerivation rec {
+  pname = "sd-mux-ctrl";
+  version = "2020-02-17";
+
+  src = fetchgit {
+    url = "https://git.tizen.org/cgit/tools/testlab/sd-mux.git";
```

It looks like they might have had a change in their cgit or git hosting config...

```suggestion
    url = "https://git.tizen.org/cgit/tools/testlab/sd-mux";
```

```
fatal: repository 'https://git.tizen.org/cgit/tools/testlab/sd-mux.git/' not found                                                     
```